inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -373,11 +374,20 @@ def prepare_files(obj, eng):
     if not obj.files:
         return
 
+    def _define_filename(paper, file_name):
+        """Define the filename in according to the article source"""
+        if is_arxiv_paper(paper):
+            return 'arxiv:' + file_name
+
+        # Default return
```

Comment:
You don't need to write these kind of comments. It's obvious that a return returns!